src/SDKs/StorSimple/Series1200/Management.StorSimple1200Series/generate.cmd

```diff
@@ -0,0 +1,7 @@
+::
+:: Microsoft Azure SDK for Net - Generate library code
+:: Copyright (C) Microsoft Corporation. All Rights Reserved.
+::
+
+@echo off
+call %~dp0..\..\..\..\tools\generate.cmd StorSimple1200Series/resource-manager %*
```

Comment:
Please replace this with a `generate.ps1` similar to [this](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Management.Compute/generate.ps1) and regenerate the code